src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf

```diff
@@ -8418,7 +8418,7 @@ Poskytněte kompilátoru nějaký způsob, jak metody rozlišit. Můžete např
         <note />
       </trans-unit>
       <trans-unit id="ERR_BadSpecialByRefLocal">
-        <source>Parameters or locals of type '{0}' cannot be declared in async methods or lambda expressions.</source>
+        <source>Parameters or locals of type '{0}' cannot be declared in async methods or async lambda expressions.</source>
         <target state="translated">Parametry nebo lokální proměnné typu {0} nemůžou být deklarované v asynchronních metodách nebo lambda výrazech.</target>
```

Comment:
It looks like you didn't run the UpdateXlf target. The state should "automatically" change to needs-review-translation.

Reply:
That's what I was hoping `git clean -xdf; .\Restore.cmd; .\Build.cmd` would cover for me. Maybe the reason it didn't do anything was because I let the replace touch .xlf files in the first place.

Reply:
> Maybe the reason it didn't do anything was because I let the replace touch .xlf files in the first place.

Yep, UpdateXlf task won't detect that this needs a review translation if it sees the source in xlf matches what's in resx.

Only when they don't match it updates it and changes the state.

Reply:
I'll get it right one of these times 😅

Reply:
This is frustrating. I undid the changes in all .xlf files and ran `.\Build.cmd` again and apparently this didn't cause the UpdateXlf target to run.

Reply:
@jnm2 This seems unexpected to me. But worth a try to run `dotnet msbuild /t:UpdateXlf` in `src\Compilers\CSharp\Portable`.

Reply:
```
C:\Users\Joseph\Source\Repos\roslyn\src\Compilers\CSharp\Portable> dotnet msbuild /t:UpdateXlf
Could not execute because the application was not found or a compatible .NET SDK is not installed.
Possible reasons for this include:
  * You intended to execute a .NET program:
      The application 'msbuild' does not exist.
  * You intended to execute a .NET SDK command:
      A compatible installed .NET SDK for global.json version [6.0.100-preview.1.21103.13] from [C:\Users\Joseph\Source\Repos\roslyn\global.json] was not found.
      Install the [6.0.100-preview.1.21103.13] .NET SDK or update [C:\Users\Joseph\Source\Repos\roslyn\global.json] with an installed .NET SDK:
        3.1.408 [C:\Program Files\dotnet\sdk]
        5.0.104 [C:\Program Files\dotnet\sdk]
        5.0.202 [C:\Program Files\dotnet\sdk]
        5.0.300-preview.21180.15 [C:\Program Files\dotnet\sdk]
```

Reply:
It's very strange that Build.cmd completed successfully while you don't have the SDK version specified in global.json. Curious to know how this occurred 👀

Reply:
I added a blank line to a .cs file in src\Compilers\CSharp\Portable and ran .\Build.cmd again. Still no xlf files being updated even though the local commit I'm on has no .xlf changes anymore.

Reply:
I'm pushing the commit I'm on just so everyone can follow along. I don't know of a way to trigger the UpdateXlf target yet without figuring out how to install the exact SDK version `6.0.100-preview.1.21103.13`.

Reply:
I'd try to change global.json to a version you already have (but dont push it) before `dotnet msbuild /t:UpdateXlf`.

I guess the reson why dotnet command fail but not build.cmd is that build.cmd uses the dotnet version from `.dotnet` directory.

Reply:
Okay, so I deleted the `sdk` key from global.json and ran `dotnet msbuild /t:UpdateXlf` in `src\Compilers\CSharp\Portable`. It didn't cause any files to change.

Reply:
Strange.
I think for now you can manually update xlfs by setting state to "needs-review-translation" and manually update the source.

I'll take a look tomorrow if I can reproduce the behavior you're seeing.

Reply:
Running `dotnet msbuild /t:UpdateXlf` worked for me, but I'm on `6.0.100-preview.2.21155.3`:

```
D:\dotnet\Roslyn\src\Compilers\CSharp\Portable>dotnet --info
.NET SDK (reflecting any global.json):
 Version:   6.0.100-preview.2.21155.3
 Commit:    1a9103db2d
```

Not sure if the issue you're encountering has something to do with the SDK version.

Opened https://github.com/jnm2/roslyn/pull/1 on your fork since I already had the changes locally.

Reply:
Thank you! That's really nice of you.